### ISSUE TITLE
epar font size too small for 4k monitor

### ISSUE THREAD

**Issue by pllim** [spacetelescope/pyraf#67]
User has HiDPI display, with 4K resolution, and has problems with PyRAF `epar` GUIs, with labels being too small. According to @cdsontag , there does not appear to be a way for user to configure font size.

**Comment by olebole** [comment]
I do not have this problem -- it seems that the Tk default were improved in the last years. It worked well at least under Linux. For Mac, I don't have a HighDPI screen to test; on 1920x1200 (a VirtualBox) it looks OK. Coould someone re-check?

**Comment by Firestar-Reimu** [comment]
Same issue on my 2560x1600 screen.

https://forum.manjaro.org/t/how-to-make-the-x-window-show-better/91646/

**Comment by olebole** [comment]
Python 2 is no longer supported. Can you try out the [release candidate 2.2.0RC1](https://pypi.org/project/pyraf/2.2.0rc1/)?

**Comment by Firestar-Reimu** [comment]
Nope, I install Pyraf 2.2.0rc1 on pip and Iraf Snapshot 2.16.1+2021.06.14

![Screenshot_20211123_200928](https://user-images.githubusercontent.com/65082432/143021645-eae3ab94-26da-4cda-b5ac-64defe023c82.png)

This is what `epar` looks like

**Comment by olebole** [comment]
You could install the `xrdb` command and run the following:
```Shell
echo Xft.dpi: 300 | xrdb -merge
```
Adjust the 300 to get the size you want. To make it permanent, put the line
```
 Xft.dpi: 300
```
into your `~/.XResources`.

**Comment by Firestar-Reimu** [comment]
```
echo Xft.dpi 192 | xrdb -merge
xrdb: colon missing on line 1, ignoring line
```

If I put the line
```
 Xft.dpi: 300
```
into my `~/.XResources`, and run `xrdb ~/.Xresources`

All the x-window become larger (for example, fcitx, as the right one is 192 dpi)

**BUT `epar` is not influenced.**

**Comment by Firestar-Reimu** [comment]
Do I need to install fonts like `xgterm*font: lucidasanstypewriter-18`

**Comment by FRIVERIN** [comment]
Hello, I am encountering the same issue. Did you find a solution?

**Comment by FRIVERIN** [comment]
Alright I'll look that up thanks
